pkg/loader/compose/compose.go

```diff
@@ -36,14 +36,27 @@ type Compose struct {
 }
 
 // load environment variables from compose file
-func loadEnvVars(e map[string]string) []kobject.EnvVar {
+func loadEnvVars(envars []string) []kobject.EnvVar {
 	envs := []kobject.EnvVar{}
-	for k, v := range e {
+	var character string
+	for _, e := range envars {
+		//FIXME:(tuna) if envvar string contains both = and :, then = will be first pick up. Consider the case: URL=http://examples.com
+		if strings.Contains(e, "=") {
+			character = "="
+		} else if strings.Contains(e, ":") {
+			character = ":"
+		} else {
+			logrus.Errorf("Invalid environment variable format, only : and = separators are supported")
```

Comment:
just variable name without `:` or `;` is also valid see: https://docs.docker.com/compose/compose-file/#/environment 
This is something that we should also consider.